ios/Classes/TiMapCustomAnnotationView.m

```diff
@@ -24,8 +24,8 @@ - (id)initWithAnnotation:(id <MKAnnotation>)annotation reuseIdentifier:(NSString
 - (void)setProxy:(TiViewProxy*)customView
 {
     if (theProxy != customView) {
-        [[theProxy view] removeFromSuperview];
-        RELEASE_TO_NIL(theProxy);
+        //[[theProxy view] removeFromSuperview];
+        //RELEASE_TO_NIL(theProxy);
```

Comment:
And this causes problems when updating the view of the annotation after it's creation, which is a very typical use-case. This was actually the first thing I tried as well 😊